src/npyffi/array.rs

```diff
@@ -394,6 +394,12 @@ pub unsafe fn PyArray_CheckExact(op: *mut PyObject) -> c_int {
     (ffi::Py_TYPE(op) == PY_ARRAY_API.get_type_object(NpyTypes::PyArray_Type)) as _
 }
 
+// these are under `#if NPY_USE_PYMEM == 1` which seems to be always defined as 1
+pub use pyo3::ffi::{
```

Comment:
I don't really understand why we need to re-export those in the first place? Why not use them directly from `pyo3`?

Reply:
(Concerning the comment, I think these would only be enabled if CPython is built for usage with e.g. Valgrind, c.f. https://github.com/python/cpython/blob/main/Misc/README.valgrind)

Reply:
I think it makes sense to re-export with new names, to match the numpy header.